<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>412</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>412</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>412</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Auto Save</string>
  </property>
  <widget class="QPushButton" name="save_btn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>400</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QSlider" name="sec_bar">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>60</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>저장 주기</string>
   </property>
  </widget>
  <widget class="QLabel" name="sec_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>초: 1</string>
   </property>
  </widget>
  <widget class="QSlider" name="min_bar">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>60</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="min_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>분: 0</string>
   </property>
  </widget>
  <widget class="QSlider" name="hour_bar">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>24</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="hour_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>시: 0</string>
   </property>
  </widget>
  <widget class="QComboBox" name="select_app">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="stop_btn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>400</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
